src/components/product/ProductCard/ProductCard.tsx

```diff
@@ -101,6 +101,7 @@ function ProductCard({
               data-value={listPrice}
               variant="listing"
               classes="text-body-small"
+              SRText="Original price:"
```

Comment:
Can we use the `kebab-case` here to follow the pattern?
```suggestion
              sr-text="Original price:"
```